migrations/V152__Update_Possible_Operational_Solution_Contacts.sql

```diff
@@ -0,0 +1,32 @@
+-- Remove Hung Van and Yolanda Villanova as Solution Contacts
+DELETE FROM possible_operational_solution_contact
+WHERE NAME IN ('Felicia Addai', 'Miyani Treva');

```

Comment:
Do you know if these contacts were a POC for any other solution, or just this one? If just this one, this looks good.